packages/cli/src/commands/deploy/index.ts

```diff
@@ -355,6 +305,66 @@ export default async (client: Client): Promise<number> => {
     throw new Error(`"org" is not defined`);
   }
 
+  // build `--prebuilt`
+  if (argv['--prebuilt']) {
+    // For repo-style linking, update `cwd` to be the Project
+    // subdirectory when `rootDirectory` setting is defined
+    if (
+      link.status === 'linked' &&
+      link.repoRoot &&
+      link.project.rootDirectory
+    ) {
+      cwd = join(cwd, link.project.rootDirectory);
+    }
```

Comment:
This is the main change. The rest is just a move of the existing code to a point after `link` is defined.